bundles/org.openhab.binding.icalendar/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -54,44 +54,44 @@
 
 	<channel-type id="event_current_title">
 		<item-type>String</item-type>
-		<label>current event title</label>
-		<description>title of a currently present event</description>
+		<label>Current event title</label>
```

Comment:
The first letter of every **word** in a label should be capitalized. Please update elsewhere as well.
```suggestion
		<label>Current Event Title</label>
```

Reply:
OK.